wire/msggetcfilters.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2017 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"io"
+
+	"github.com/roasbeef/btcd/chaincfg/chainhash"
+)
+
+// MaxGetCFiltersReqRange the maximum number of filters that may be requested in
+// a getcfheaders message.
+const MaxGetCFiltersReqRange = 100
```

Comment:
I think this should actually be 1000, as within the BIP we allow a spacing of no more than 1000: 
> The height of the block with hash StopHash MUST be greater than or equal to StartHeight, and the difference MUST be strictly less than 1,000.

Reply:
Yeah, the spec says 1,000, but as I was writing this I realized 100 might be a more sane value given the relative size of filters to filter headers. What do you think? If you agree, I think it's still fine to edit the spec.